[rocr-debug-agent] Support % format tokens in -o and --save-code-objects (#7707)
ROCm/rocm-systems: projects/rocr-debug-agent/docs/how-to/user-guide.rst

  * - ``-s [DIR]``, ``--save-code-objects[=DIR]``
    - Saves all loaded code objects. If the directory is not specified, the
      code objects are saved in the current directory. Directory name can

>       contain ``%`` format tokens which will be expanded as specified below.

Reviewer: ```suggestion
      contain ``%`` format tokens which will be expanded as specified below:
```

Reviewer 2: Not sure about this one.  The description of the available options is later in the text (after the table), to me the ":" would imply the description comes immediately.

Reviewer: Then let's mention as specified in the following ......(section)?

Reviewer 2: done.

      compatible with ``-c``.

  * - ``-o <file-path>``, ``--output=<file-path>``
    - Saves the output produced by the ROCdebug-agent in the specified file.

>       The file-path can contain '%' format tockens which will be expanded

Reviewer: ```suggestion
      The file path can contain '%' format tokens, which will be expanded
```

>       according to the specification below.

Reviewer: ```suggestion
      according to the following specification:
```

Reviewer 2: same, the ":" would seem odd here.

  * - ``-h``, ``--help``
    - Displays a usage message and aborts the process.

Supported ``%`` format tokens:
  * ``%p``: process ID of the application being debuggged
  * ``%h``: host name

>   * ``%t``: timestand (seconds since the Epoch) of when the patch is

Reviewer: ```suggestion
  * ``%t``: timestamp (seconds since the Epoch) when the patch is
```

>     expaded

Reviewer: ```suggestion
    expanded
```

>   * ``%e``: short name of the of the application being debugged

Reviewer: ```suggestion
  * ``%e``: abbreviated name of the application being debugged
```
